lib/alerts/parser.ex

```diff
@@ -10,17 +10,20 @@ defmodule Alerts.Parser do
     def parse(%JsonApi.Item{type: "alert", id: id, attributes: attributes}) do
       Alerts.Alert.new(
         id: id,
-        header: attributes["header"],
-        informed_entity: parse_informed_entity(attributes["informed_entity"]),
         active_period: Enum.map(attributes["active_period"], &active_period/1),
-        effect: effect(attributes),
+        banner: description(attributes["banner"]),
         cause: cause(attributes["cause"]),
-        severity: severity(attributes["severity"]),
+        created_at: parse_time(attributes["created_at"]),
+        description: description(attributes["description"]),
+        effect: effect(attributes),
+        header: attributes["header"],
+        image: attributes["image"],
+        image_alternative_text: attributes["image_alternative_text"],
```

Comment:
📓 `created_at`, `image`, and `image_alternative_text` are the new alert fields we are using in this feature. Everything else is just alphabetical re-arrangement.